index.html

```diff
@@ -1,18 +1,16 @@
 
-<!DOCTYPE html>
-<html>
-<head lang="en">
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>vaadin-icons</title>
-
-  <script src="../elements-demo-resources/ga.js"></script>
```

Comment:
this line seems to be the only meaningful change for this file

Reply:
Yes. In addition to removing that unnecessary script I just wanted the code style/formatting of this file to match the skeleton (and other components based on it).

Both of these changes were done in separate commits.